rs-matter-macros/src/lib.rs

```diff
@@ -468,3 +472,41 @@ pub fn derive_fromtlv(item: TokenStream) -> TokenStream {
         )
     }
 }
+
+#[derive(Debug)]
+struct MatterIdlImportArgs {
+    path: String,
+}
+
+impl Parse for MatterIdlImportArgs {
+    fn parse(input: syn::parse::ParseStream) -> syn::Result<Self> {
+        let path: syn::LitStr = input.parse()?;
+
+        Ok(MatterIdlImportArgs { path: path.value() })
+    }
+}
+
+#[proc_macro]
+pub fn matter_idl_import(item: TokenStream) -> TokenStream {
```

Comment:
Another nit but could not resist: how about abbreviating this to just `idl_import`? `matter` is kind of unnecessary, as the macro would live in the `rs_matter_macros` crate namespace anyway.

Reply:
Sure, matter is responsible for a lot of smurf naming here. Updated.